### ISSUE TITLE
Should we add Glosario to Jargon Busting?

### ISSUE THREAD

**Issue by libcce** [LibraryCarpentry/lc-overview#28]
This resource was recently introduced via The Carpentries: https://carpentries.github.io/glosario/

Is it useful for the Jargon Busting activity, as another resource to mention?

**Comment by ragamouf** [comment]
It could be! It's a work in progress, so the FIXMEs might be red flags. Because the remit is a glossary of definitions, some of which are undefined, I will optimistically hope that the overall goal is list of *working* definitions, rather than ontological debate.

**Comment by morskyjezek** [comment]
Yes, this seems useful and has remained in use. I'll work on adding this to the page.
